sys/kern/subr_intr.c

```diff
@@ -100,12 +110,17 @@ MALLOC_DECLARE(M_INTRNG);
 MALLOC_DEFINE(M_INTRNG, "intr", "intr interrupt handling");
 
 /* Main interrupt handler called from assembler -> 'hidden' for C code. */
+void intr_irq_handler_root(struct trapframe *tf, uint32_t rootnum);
```

Comment:
`style` just says all functions are prototyped somewhere. I don't see the point in adding `extern` before the function since they default to external linkage anyway, but regardless this is irrelevant since I had to move the declaration to sys/intr.h.

Reply:
I had missed this was due to RISC-V calling `intr_irq_handler()` from C.  That invalidates my concern.